go/vt/srvtopo/resilient_server_test.go

```diff
@@ -404,7 +403,7 @@ func TestGetSrvKeyspaceCreated(t *testing.T) {
 }
 
 func TestWatchSrvVSchema(t *testing.T) {
-	watchSrvVSchemaSleepTime = 10 * time.Millisecond
```

Comment:
I see this wasn't being used anywhere (and the test was using 10ms explicitly everywhere).
Do you feel like replacing 10ms with this var and 5ms with another one (or this one / 2)?
While you are at it, it may also be worth replacing `t.Fatalf` with `require`.